### PR TITLE
EN-3717, fixed timer function, moved it outside to common utils

### PR DESCRIPTION
**Pull request by nitzanpx**

### CODE REVIEW

File: lib/px/utils/pxcommonutils.lua [hunk 1]
```diff
@@ -0,0 +1,21 @@
+--
```
Comment (yaronpx) [inline lib/px/utils/pxcommonutils.lua]: why do we need this file? why didn't you leave it in `pxtimer`?

Comment (nitzanpx) [inline lib/px/utils/pxcommonutils.lua]: @yaronpx pxtimer uses to measure time outside the application scope, adding more logic to this file was not a good practice to implement it there

once it was crashing I didnt want to start and modify this file, I decided to move it to a util class so later we can add more utils there

**Comment by nitzanpx** [comment]
@yaronpx  lets talk about this if the answer doesn't satisfy you

File: lib/px/utils/pxcommonutils.lua [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+-- Created by IntelliJ IDEA.
+-- User: nitzangoldfeder
+-- Date: 16/06/2017
+-- Time: 12:17
+-- To change this template use File | Settings | File Templates.
+--
```
Comment (bendpx) [inline lib/px/utils/pxcommonutils.lua]: hey, please avoid pushing these auto-generated headers ^^ to the new files.

**Pull request merged**